builtin-iterator-zip.js

```diff
@@ -0,0 +1,210 @@
+;(function(IteratorHelper, InternalError, TypeError, call, Symbol·iterator) {
```

Comment:
A thing to keep in mind when reviewing this file: it's almost impossible to change or reorder anything here without breaking some test262 tests.